crates/bevy_window/src/window.rs

```diff
@@ -602,12 +602,26 @@ impl Window {
     }
 }
 
+/// Describes the information needed for a window creation like width, height, title, present mode, etc.
+/// This should be set up before adding the [`WindowPlugin`]. Most of these settings can also later
+/// be configured through the [`Window`] resource.
+///
+/// See `examples/window/window_settings.rs` for usage.
+///
+/// [`Window`]: crate::Window
+/// [`WindowPlugin`]: crate::WindowPlugin
 #[derive(Debug, Clone)]
 pub struct WindowDescriptor {
+    /// The requested logical width of the window's client area.
```

Comment:
What does "logical width" mean more precisely? Is this pixels? pixels modified by the scale?

Reply:
None of the other supporting classes mentions what it is about but the ```Window``` class has a full description of the sizes - logical and physical. I just wanted to mention here that it is logical size and not bother with copying the whole section out of the ```Window``` class.

Reply:
Added one more line of description why it might vary.